### ISSUE TITLE
Accessibility page minor tweaks 

### ISSUE THREAD

**Issue by joshuadkitenge** [ral-facilities/scigateway#986]
**Description**:
Addressing minor tweaks to the accessibility page

**Acceptance criteria**:

- [ ] Contact email should render as a mailto link 
- [ ] The text under bullet points in the "Non-compliance with the accessibility regulations" section should be changed to give rough timing
